sklearn/linear_model/_bayes.py

```diff
@@ -15,7 +15,8 @@
 from ..utils.extmath import fast_logdet
 from scipy.linalg import pinvh
 from ..utils.validation import _check_sample_weight
-from ..utils._param_validation import Interval
+from ..utils._param_validation import Interval, Hidden, StrOptions
+import warnings
```

Comment:
can you move this import on the top since this is part of the standard Pyhon library

Reply:
Sure.I will do that in the next commit.